src/styles/elements/_header.scss

```diff
@@ -0,0 +1,82 @@
+.header {
+  width: 100%;
+  display: flex;
+  justify-content: space-between;
+  align-items: center;
+
+  &__logo {
+    margin-left: 50px;
+    position: relative;
+    width: 40px;
+    height: 40px;
+  }
+
+  &__image {
+    position: absolute;
+    top: 0;
+    left: 0;
+    width: 100%;
+    height: 100%;
+  }
+}
+
+.nav {
```

Comment:
I'd say nav is a separate block. So, according to BEM rules it should be in a separate file